shaka/src/public/ShakaPlayerView.mm

```diff
@@ -121,6 +122,12 @@ - (void)setup {
 
   _gravity = shaka::VideoFillMode::MaintainRatio;
   _cues = [[NSMutableDictionary alloc] init];
+
+  [[UIDevice currentDevice] beginGeneratingDeviceOrientationNotifications];
+  [[NSNotificationCenter defaultCenter]
+     addObserver:self selector:@selector(orientationChanged:)
+     name:UIDeviceOrientationDidChangeNotification
+     object:[UIDevice currentDevice]];
```

Comment:
This doesn't match our style.  You can use `clang-format` to format this file.  Or you can just copy the indentation from other parts.  The first `addObserver:` part should be on the same line and following args should be aligned with the `:`.

Reply:
Done